db/model/aspect.js

```diff
@@ -341,6 +353,30 @@ module.exports = function aspect(seq, dataTypes) {
             resolve(found.length === 1);
           }));
       }, // isWritableBy
+
+      checkGeneratorReferences(action) {
+        const aspectName = this.previous('name');
+
+        // the aspect names are stored in lowercase on the generator
+        const where = { aspects: { $contains: [aspectName.toLowerCase()] } };
+        return seq.models.Generator.findAll({ where })
```

Comment:
Was the lower casing done just to use `contains` instead of `like`?

Reply:
This queries on the Generator's "aspects" field, which is an array, so like doesn't work. The lower casing was done because it was the only way I found to make the query case-insensitive without setting up an association or iterating over all generators.

Reply:
Makes sense.